smriprep/workflows/anatomical.py

```diff
@@ -425,8 +421,9 @@ def _check_img(img):
             (buffernode, t1w_dseg, [('t1w_brain', 'in_files')]),
             (t1w_dseg, lut_t1w_dseg, [('tissue_class_map', 'in_dseg')]),
             (t1w_dseg, anat_norm_wf, [
-                ('probability_maps', 'inputnode.moving_tpms')]),
-            (t1w_dseg, outputnode, [('probability_maps', 't1w_tpms')]),
+                (('probability_maps', _probseg_fast2bids), 'inputnode.moving_tpms')]),
```

Comment:
This reorders the probseg files for bids. It is relevant because this is the easiest way of propagating them, and assigning them the correct `label-` by the DerivativesDataSink.